flink-kubernetes/src/main/java/org/apache/flink/kubernetes/kubeclient/parameters/KubernetesJobManagerParameters.java

```diff
@@ -160,4 +160,18 @@ public String getEntrypointClass() {
     public boolean isInternalServiceEnabled() {
         return !HighAvailabilityMode.isHighAvailabilityModeActivated(flinkConfig);
     }
+
+    public int getReplicas() {
+        final int replicas =
+                flinkConfig.get(KubernetesConfigOptions.KUBERNETES_JOBMANAGER_REPLICAS);
+        checkArgument(
+                replicas > 0,
+                "'%s' should not be configured less than one.",
+                KubernetesConfigOptions.KUBERNETES_JOBMANAGER_REPLICAS.key());
+        if (replicas > 1 && !HighAvailabilityMode.isHighAvailabilityModeActivated(flinkConfig)) {
+            throw new IllegalArgumentException(
+                    "High availability should be enabled when starting standby JobManagers.");
+        }
```

Comment:
`IllegalConfigurationException` is preferred rather than `IllegalArgumentException`.